.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberRangeParser.cs

```diff
@@ -69,8 +69,8 @@ private ParseResult ParseNumberRangeWhichHasTwoNum(ExtractResult extResult)
                 endValue = nums[0];
             }
 
-            var startValueStr = Config.CultureInfo != null ? startValue.ToString(Config.CultureInfo) : startValue.ToString();
-            var endValueStr = Config.CultureInfo != null ? endValue.ToString(Config.CultureInfo) : endValue.ToString();
+            var startValueStr = startValue.ToString();
```

Comment:
Isn't it better to specify InvariantCulture? This will avoid issues with the local settings on the machine where the code runs.

Reply:
Thank you for your suggestion. It is very helpful. I will fix this in next iteration.